stdlib/2/urlparse.pyi

```diff
@@ -1,8 +1,9 @@
 # Stubs for urlparse (Python 2)
 
-from typing import AnyStr, Dict, List, NamedTuple, Tuple, Sequence, Union, overload, Optional
+from typing import AnyStr, Dict, List, NamedTuple, Tuple, TypeVar, Sequence, Union, overload, Optional
 
 _String = Union[str, unicode]
+_AnyText = TypeVar('_AnyText', str, bytes, unicode)
```

Comment:
This is just `typing.AnyStr` (str and bytes are the same in Python 2).

Reply:
Corrected in 003dd638.